ui.apps/src/main/content/jcr_root/apps/acs-commons/config/com.adobe.acs.commons.users.impl.EnsureServiceUser-email-service.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0" 
+jcr:primaryType="sling:OsgiConfig" 
+principalName="acs-commons/acs-commons-email-service" 
+operation="add" 
+ensure-immediately="{Boolean}true" 
+aces="[type=allow;privileges=jcr:read;path=/conf/global]" />
```

Comment:
I tested this on a fresh instance of AEM 6.4.4 and on deployment I do not see the permissions added to /conf/global.  Can't say why - the config looks correct and I even see it on the configMgr felix interface.

The other existing EnsureServiceUser configs seem to be adding their permissions just fine, the service itself is working fine.

I tried restarting AEM to see if the permissions would be picked up on restart, and I still see no rep:policy on /conf/global.

Could the issue be that there is no rep:policy below /conf/global in the first place (on a clean server)?

Reply:
I removed the user from PRINCIPAL_NAMES and it should be good now. :)